### PR TITLE
[Docs]: Added 'system config' as a mesheryctl command

### LINKED ISSUE #1168: [Docs] Add `system config` as new `mesheryctl` command
**Current State:**
The https://meshery.layer5.io/docs/guides/mesheryctl doc does not include the new `mesheryctl system config` command created in PR https://github.com/layer5io/meshery/pull/1084

**Desired State:**
The docs should include this new command along with example usage.

---
**Contributor Resources**
- [Meshery documentation site](https://meshery.layer5.io/docs/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by chandrashritii**
Signed-off-by: Shriti <shritichan@gmail.com>

This PR will update the meshery/docs to show system config as a mesheryctl command. Fixes #1168 

#### Checklist

- [x] [Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin).
- [x] PR title includes `[<name of area>]:<description>`

### CODE REVIEW

**Comment by ashiscs** [comment]
fixes #1168

File: docs/pages/guides/mesheryctl.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ type: guide
 |                   | version             | Displays the version of the Meshery Client (`mesheryctl`) and the SHA of the release binary. | `mesheryctl system version` |
 |                   | help                | Displays help about any command. | `mesheryctl --help` |
 |                   | --mesheryctl-config | (optional) path to Meshery Client (`mesheryctl`) configuration file (`~/.meshery/mesheryctl.yaml`) to overrides defaults. | `mesheryctl perf <args> --mesheryctl-config=~/.meshery/mesheryctl.yaml` |
+|                   | --system config     | configures Meshery with the kubeconfig, generated with the help of user details, to provide cluster access for public clouds(GKE/EKS). | `./meshery  system config gke --token "PATH TO TOKEN"` |
```
Comment (leecalcote) [inline docs/pages/guides/mesheryctl.md:16]: @chandrashritii you may want to update `./meshery  system config gke --token "PATH TO TOKEN"` to remove `./` and add `ctl`, like so: `mesheryctl  system config gke --token "PATH TO TOKEN"`

Comment (chandrashritii) [reply docs/pages/guides/mesheryctl.md]: @leecalcote The change has been made in the latest commit

**Comment by leecalcote** [comment]
Thank you, @ashiscs. @chandrashritii, thanks for this. This gets the job done. As you become more familiar with the project, you may want to revisit the description used here. I don't have a succinct suggestion to give you at the moment.

Given this new feature, we can take much of what is on this page https://meshery.layer5.io/docs/installation/gke#quick-start-with-google-kubernetes-engine-gke and the EKS page and move these instructions into a "Manual Configuration" section. Above the "Manual Configuration" section, you can regurgitate the description of this `mesheryctl system config` function and its use.

**Comment by leecalcote** [comment]
Haha! Yay, @chandrashritii 🎈

**Pull request merged**